contents/docs/_snippets/identify-intro.mdx

```diff
@@ -0,0 +1,27 @@
+Linking events to specific users enables you to gain full insights as to how they're using your product across different sessions, devices, and platforms. 
+
+This is straight forward to do when [capturing backend events](/docs/getting-started/send-events#3-capture-backend-events), as you associate events to a specific user using `distinct_id`, which is a required argument. However, on the [frontend]((/docs/getting-started/send-events#sending-custom-properties-on-an-event)), a `distinct_id` is not a required argument and thus events can be submitted anonymously. To link events from anonymous to specific users, you need to use `identify`:
+
+<MultiLanguage selector="tabs">
+
+```javascript
+posthog.identify(
+    'distinct_id',
+    { email: 'max@hedgehogmail.com', name: 'Max Hedgehog' } 
+    {}, 
+);
+```
+
+```Android
+PostHog.with(this)
+       .identify(distinctID, new Properties()
+                                .putValue("name", "My Name")
+                                .putValue("email", "user@posthog.com"));
+```
```

Comment:
```suggestion
```Android
PostHog.with(this)
     .identify(distinctID, new Properties()
        .putValue("name", "My Name")
        .putValue("email", "user@posthog.com"));
```
```